Big-Spender/readme.md

```diff
@@ -124,7 +124,10 @@ CREATE YOUR QUERY HERE
 **You:** Sure thing. To confirm, the date is August 19, 2021, the transaction number is 38104091, the supplier invoice number is 3780119655, the supplier is 'Dell', the expense type is 'Hardware' and the expense area is 'IT'. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERIES HERE
+insert into suppliers (supplier) values('Dell');
+insert into expense_types (expense_type) values ('Hardware');
+insert into expense_areas(expense_area)values('IT');
+insert into spends (expense_type_id, expense_area_id, supplier_id, date, transaction_no, supplier_inv_no, description, amount) values (42, 46, 66, '2021-08-19', 38104091, 3780119655,'Computer Hardware Dell', 32000)
```

Comment:
great job! Nice one on these!

Reply:
thank you so much for reviewing my code Karley! I've made some changes as you suggested :)